app/components/kpi/component.js

```diff
@@ -0,0 +1,93 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import { format, getDaysInMonth } from 'date-fns';
+import { action } from '@ember/object';
+
+export default class KpiComponent extends Component {
+  @service store;
+  @service session;
+
+  get tableOfMonths() {
+    return Array.from({ length: 12 }, (_, index) =>
+      format(new Date(2022, index), 'MMMM')
+    );
+  }
+
+  get allClosedRequests() {
+    return this.args.model.filter(({ status }) => status === 'closed');
+  }
+
+  get requestTypes() {
+    return [...new Set(this.args.model.map(({ type }) => type))];
+  }
+
+  get machinesName() {
+    return [...new Set(this.args.model.map(({ machine }) => machine))];
+  }
+
+  @action
+  tableOfMonthsWithRequests(requestType) {
+    return this.tableOfMonths.map(
+      (month, index) =>
+        this.allClosedRequests
+          .filter(({ createdAt }) => createdAt.getMonth() === index)
+          .filter(({ type }) => type === requestType).length
```

Comment:
It's ok, but for better performance consider to do it like this:
```suggestion
        this.allClosedRequests
          .filter(({ createdAt, type }) => createdAt.getMonth() === index && type === requestType).length
```